widgets/views/templateSelectModal.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use humhub\widgets\Button;
+use humhub\widgets\Modal;
+use humhub\widgets\ModalButton;
+use yii\helpers\Html;
+use humhub\modules\wiki\helpers\Url;
+use yii\widgets\ActiveForm;
+
+$templateOptions = [];
+
+foreach ($templates as $template) {
+    $templateOptions[$template->id] = Html::encode($template->title);
+}
+
+?>
+
+<?= Modal::widget([
```

Comment:
If possible, modal should just show up, if there are any templates defined so far.
If not template is defined, just a blank page should be created.